compose-builder/common-sec-stage-gate.env

```diff
@@ -39,3 +39,4 @@ STAGEGATE_READY_TORUNPORT=54329
 PROXY_SETUP_HOST=edgex-proxy-setup
 API_GATEWAY_HOST=kong
 API_GATEWAY_STATUS_PORT=8001
+ADD_PROXY_ROUTE=admin-api.http://localhost:8001
```

Comment:
Does this env needs to be shared among other containers?  If not, add it under `environment` section of that specific container in the know.

Reply:
Yea, what service need this?

Reply:
Well, I put it under there because that environment file is specific to security - this implements the security loopback. However, after talking with @bnevis-i yesterday, I found some other options, and I am in the middle of a different implementation right now. I may end up backing this PR out altogether in favor of an implementation that provides better separation.